### PR TITLE
Add SSVC python module

### LINKED ISSUE #325: Create python package to facilitate json round-tripping
Blocked by
- [x] #321 

Fill in more detail as needed, but at least:

- use `dataclasses_json` + `dataclasses` for the objects
- be able to generate json examples from python code to incorporate into reference documentation
- python installer for package using `pyproject.toml` https://pip.pypa.io/en/stable/reference/build-system/pyproject-toml/
- include code documentation in the reference docs as well

### PR DESCRIPTION
**Pull request by ahouseholder**
Represents SSVC v1, v2, v2.1 decision point groups and decision points as python dataclasses using the dataclasses-json module to allow for json format data roundtrip.

Includes schema and schema validation tests.

Note: this branch includes the schemas defined in PR #340, so we should resolve that PR before this one.

- blocked by #340 
- resolves #325 

This PR is part 1 of N (where N is probably 2-4) that breaks up #339.
Future PRs will likely address #319 and #330, as well as add generated documentation based on the modules in this PR.  I am deliberately keeping this PR limited to python changes necessary to implement a resolution to #325

### CODE REVIEW

File: data/schema/Decision_Point.schema.json
Comment (sei-vsarvepalli) [inline data/schema/Decision_Point.schema.json:1]: We need to update the $id field to point to the new URL - 
https://certcc.github.io/SSVC/data/schema/Decision_Point.schema.json

File: data/schema/Decision_Point_Group.schema.json
Comment (sei-vsarvepalli) [inline data/schema/Decision_Point_Group.schema.json:1]: We need to update the $id field to point to the new URL - 
https://certcc.github.io/SSVC/data/schema/Decision_Point_Group.schema.json

**Review by sei-vsarvepalli — CHANGES_REQUESTED** [review]
Nice! The test cases are nice to have. Just need to some small updates to the JSON schema as they are actually been moved /data/schema/ path.

File: data/schema/Decision_Point.schema.json
Comment (ahouseholder) [reply data/schema/Decision_Point.schema.json:1]: That should happen in #340 though right?

Comment (ahouseholder) [reply data/schema/Decision_Point.schema.json:1]: Addressed in https://github.com/CERTCC/SSVC/pull/340/commits/554feb3b2afb5afee8567401e34761d0cf400f5d

File: data/schema/Decision_Point_Group.schema.json
Comment (ahouseholder) [reply data/schema/Decision_Point_Group.schema.json:1]: Addressed in https://github.com/CERTCC/SSVC/pull/340/commits/eb73c380198af402c6959303bc0e80a201e319e2

**Review by sei-vsarvepalli — APPROVED** [review]

**Pull request merged**